satpy/composites/__init__.py

```diff
@@ -1555,3 +1555,34 @@ def __call__(self, projectables, nonprojectables=None, **info):
 
         masked_projectable = projectable.where(lon_min_max)
         return super(LongitudeMaskingCompositor, self).__call__([masked_projectable], **info)
+
+class MSGQuicklook(GenericCompositor):
```

Comment:
I'd say the compositor name should be something less generic, even if this is what the authors in the reference call it. Maybe `MSGPseudoTrueColor`, `MSGSimulatedTrueColor`, or something. Or if it could be adapted to other instruments, drop `MSG`.

Reply:
Ok, I'll ask the author of the article about the "official name".

Reply:
Did you get any answer for this?

Reply:
Yes, I wrote to one of the authors
he told me that he didn't have an official name, but in the article it says "RGB synthetic image" (https://www.frontiersin.org/articles/10.3389/frsen.2021.666516/full)
so the name "Synthetic True Color", for whom I wrote, seems fine to him

Reply:
Ok, thanks! could you rename the class accordingly?

Reply:
@mraspaud, done name changed :)

Reply:
Thanks!